helm/hpcc/templates/_helpers.tpl

```diff
@@ -905,6 +928,7 @@ Pass in dict with root and me
 {{- $serviceName := printf "sasha-%s" .me.name -}}
 {{- if .me.storage }}
 {{ include "hpcc.addVolume" (dict "root" .root "name" $serviceName "me" .me.storage) -}}
+{{ include "hpcc.addVolumes" (dict "root" .root "includeLabels" (list "sasha") ) }}
```

Comment:
prior to this change, each sasha service might only mount 1 pvc, now using planes and "sasha" label, it would mount all, which would be a problem if ReadWriteOnce type.

If the/a, service names a plane, and it's a pvc, I think should only mount that one